zilliqa/src/consensus.rs

```diff
@@ -746,6 +820,14 @@ impl Consensus {
                 "storing vote for new view"
             );
             if supermajority {
+                if self.view < new_view.view {
```

Comment:
This allows you to potentially catch up if your view is less but you see everyone agreed on a higher view